README.md

```diff
@@ -6,6 +6,8 @@
 
 - We don't want a one-off fix, we want to deepen our understanding of the problem space
 
+- We want to give people a resource where they can benefit from our experience without taking up our time
```

Comment:
I think I get how you mean this, but I believe it could be read in the wrong way. Maybe:
`We want to give people a resource where they can benefit from our experiences and allows us to focus on other challenges instead of the same reoccurring problems`

Reply:
Yeah, it's really hard to phrase without being a mouthful, isn't it?

Reply:
I agree that yours was more concise... and correctly worded: "allows" :stuck_out_tongue_winking_eye:  (fixed that now)

I just think "benefit from our experience without taking up our time" seems a bit harsh and while it's a message that might be good for the small set of writers, it might make others feel like we are talking down to them. But yeah, I had trouble phrasing it in a way that's concise.